src/partials/image-component.html

```diff
@@ -1 +1,13 @@
-<div class="image-block"><img {{> image-attributes image=image}}/></div>
\ No newline at end of file
+<div class="image-block">
+	{{#if image.url-webp}}
+		<picture>
+			<source srcset="{{image.url-webp}}" type="image/webp">
+		
+		<img src="{{image.url}}"{{#if image.url-set}} srcset="{{image.url-set}}"{{/if}}{{#if image.title}} title="{{image.title}}"{{#unless image.alt}} alt="{{image.title}}"{{/unless}}{{/if}}{{#if image.alt}} alt="{{image.alt}}"{{/if}} />
```

Comment:
if there is no alt-Text, there should be a fallback to an empty alt-Attribute

Reply:
What exactly should be the Fallback? If there is no alt text available there is no value of any kind of string? Or you mean not adding an alt attribute at all in this case?

Reply:
if theres no image.alt and no image.title -> `alt=""` this is the Fallback

Reply:
Understand, is done, see latest commit